x/evm/keeper/rpc_client.go

```diff
@@ -108,7 +108,7 @@ type PrepareTxEVMConfig struct {
 	EvmDenom  string
 	ExtraEips []int
 	// *rpctypes.StateOverride : original type
-	Overrides string
+	Overrides []byte
```

Comment:
If we switch to gRPC, we should switch everything to gRPC. This means all these are Protobuf, and the SGX runs a gRPC server. I think it's also cleaner in general.

Reply:
@amaury1093 Gotcha.